entity-framework/core/providers/sqlite/value-generation.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SQLite Database Provider - Value Generation - EF Core
+description: Value Generation Patterns Specific to the SQLite Entity Framework Core Database Provider
+author: AndriySvyryd
+ms.date: 09/26/2025
+uid: core/providers/sqlite/value-generation
+---
+# SQLite Value Generation
+
+This page details value generation configuration and patterns that are specific to the SQLite provider. It's recommended to first read [the general page on value generation](xref:core/modeling/generated-properties).
+
+## AUTOINCREMENT columns
+
+By convention, numeric primary key columns that are configured to have their values generated on add are set up with SQLite's AUTOINCREMENT feature. Starting with EF Core 10, SQLite AUTOINCREMENT is a first-class feature with full support through conventions and the Fluent API.
```

Comment:
Link to [the SQLite doc page](https://sqlite.org/autoinc.html) explaining what AUTOINCREMENT is.

I'm not sure what the 2nd sentence exactly means for the user... I'd maybe instead have a small Limitations sections at the bottom of the page listing the concrete problems that existed before EF 10, but as it is this sentence doesn't actually seem to tell the user anything concrete.